spec/validator_spec.rb

```diff
@@ -577,11 +577,7 @@
         :body => File.read(File.join(File.dirname(__FILE__),'..','features','fixtures','valid.csv')))
     validator = Csvlint::Validator.new("http://example.com/example.csv")
     expect( validator.valid? ).to eql(true)
-    data = validator.data
-
-    expect( data.count ).to eql 3
-    expect( data[0] ).to eql ['Foo','Bar','Baz']
```

Comment:
I see why they've added the `data` property now. I guess it would have been useful if they'd just left it behind some flag saying it's only populated when we're unit-testing?

Reply:
I saw this test which uses the data variable but do we need to check this ? are you suggesting to introduce a flag to toggle this storing of rows and bring back these tests?

Reply:
I'm not that fussed about doing it now. I don't think it's that useful for us. But I think that's the approach that should have been taken when the `@data` attribute was added.

Reply:
Yeah, definitely